WalletWasabi.Fluent/Views/CoinSelection/Columns/LabelsView.axaml

```diff
@@ -0,0 +1,12 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+             xmlns:columns="clr-namespace:WalletWasabi.Fluent.ViewModels.CoinSelection.Columns"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="WalletWasabi.Fluent.Views.CoinSelection.Columns.LabelsView"
+             x:DataType="columns:LabelsViewModel"
+             x:CompileBindings="True">
+  <controls:TagsBox IsReadOnly="True" Items="{Binding SmartLabel}" />
```

Comment:
Please use `LabelsItemsPresenter`  is in other TreeDataGrid's as it much more efficient and designed for lists.

https://github.com/zkSNACKs/WalletWasabi/blob/0eb19d33af8e56f1621e28032ce646bc43b2b629/WalletWasabi.Fluent/Views/Wallets/Home/History/Columns/LabelsColumnView.axaml#L13-L15

Reply:
Thank you! It's better now! :)